src/lib/Libifl/pbs_ifl_wrap.c

```diff
@@ -772,166 +804,33 @@ SWIG_Python_str_AsChar(PyObject *str)
 #if PY_VERSION_HEX >= 0x03000000
 #  define SWIG_Python_str_DelForPy3(x) free( (void*) (x) )
 #else
-#  define SWIG_Python_str_DelForPy3(x)
+#  define SWIG_Python_str_DelForPy3(x) 
```

Comment:
More training whitespace. If this file was generated by swig we should check the source file that is used to generate it. If this is a swig-ism then I think it's OK to leave it.

Reply:
I verified the swig file, It has trailing whitespaces.